Cargo.toml

```diff
@@ -29,6 +30,7 @@ anyhow = "1"
 cfg-if = "1"
 enum-iterator = "0"
 getset = "0"
+git-repository = { version = "0.16.0", optional = true, default-features = false, git = "https://github.com/Byron/gitoxide", branch = "main" }
```

Comment:
Note that his is only for the time being - when it's clear this PR should go ahead it will be switched to the published version on crates.io.